recipes/cpu_features/all/conanfile.py

```diff
@@ -47,10 +43,9 @@ def config_options(self):
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
-        cmake = CMake(self)
-        cmake.definitions["BUILD_PIC"] = self.options.get_safe("fPIC", True)
-        cmake.configure()
-        self._cmake = cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_PIC"] = self.options.get_safe("fPIC", True)
+        self._cmake.configure() # Does not support out of source builds
```

Comment:
Is this comment correct?
Didn't it build properly before?

If a project is buildable as a subproject, doesn't it also support out-of-source builds?
Or what problem am I missing?

Reply:
It does not build (see build 6) I was pressed on time and did not look into it. It fails to find files so I assume theres some hard coded module path.

sub project does not guarantee out of source build

Reply:
[Build 6 hit an internal unknown error]https://github.com/conan-io/conan-center-index/pull/5960#issuecomment-864331522) and its logs are not available: https://c3i.jfrog.io/c3i/misc/logs/pr/5960/

Btw, what does `X-tapaholes` mean?

Reply:
> Btw, what does `X-tapaholes` mean?

You are digging too deep into our secrets 😄   We are in the process of activating a new configuration to cross-build binaries from regular Macos to M1, at this moment, this process consist on two different actions:
 * on one hand, I trigger a build that traverses the graph taking into account the dependency order and trying to build these new binaries... it comes across different failures and I try to fix them manually (I'm sure you have seen me opening some PRs during these days)
 * but at the same time, we try to build M1 binaries in the PRs. After a successful build, we trigger another one (`X-tapaholes`) including the Macos-armv8 configurations just in case it succeeds (check report in the `summary.json` file). This build is not reported to users and _not visible_.

We want to improve this process and report the result to users, maybe some of our power users want to fix these errors even if those configurations are not required in order to merge the PR. Probably the way to go is to show more status checks with the proper links.

Reply:
Your recent M1 related pr's are indeed not ignorable :)

Great to hear about the progress and work that's being put in c3i!
Too bad a lot of this work is hidden from plain sight. It's valued very much.

Your idea about adding M1 build fails as a non-critical error to the conan-center-bot logs looks like a good idea to me.

[But, this still doesn't explain what `tapaholes` mean](https://translate.google.be/?sl=es&tl=en&text=tapaholes&op=translate) :smile:

Reply:
LOL. Long time ago we created this `Tapaholes` job to trigger a build of a reference already merged in `master` branch. The main objective was to build missing packages. By that time we said that the purpose was to _fill in the gaps_. In Spanish we say "tapar agujeros" (to cover the holes), so just putting it together: **tapa-holes**... and the rest is history 😄

Reply:
I typed 6 but I meant 5 

```
cpu_features/0.6.0: 
CMake Error at CMakeLists.txt:6 (include):
  include could not find load file:

    /Users/jenkins/w/BuildSingleReference/.conan/data/cpu_features/0.6.0/_/_/build/647afeb69d3b0a2d3d316e80b24d38c714cc6900/build_subfolder/conanbuildinfo.cmake


CMake Error at CMakeLists.txt:7 (conan_basic_setup):
  Unknown CMake command "conan_basic_setup".
```